examples/ping/src/generic_checksum.adb

```diff
@@ -20,7 +20,7 @@ is
    is
       use type ICMP.Checksum;
       use type Types.Index;
-      Checksum : ICMP.Checksum := Shl (ICMP.Checksum (ICMP.To_Base (Tag)))
+      Checksum : ICMP.Checksum := Shift_Left (ICMP.Checksum (ICMP.To_Base (Tag)))
                                       + ICMP.Checksum (ICMP.To_Base (Code));
```

Comment:
Wrong indentation.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/355/commits/8b1e9fff601aee7cdd2046e741b4c26cc48a30c5.